system/src/wifitester.cpp

```diff
@@ -156,23 +156,36 @@ void WiFiTester::printInfo() {
 }
 
 #if WIFI_SCAN
-void wlan_scan_callback(void* data, const uint8_t* ssid, unsigned ssid_len, int rssi)
+void wlan_scan_callback(void* data, const uint8_t* ssid, unsigned ssid_len, int rssi, const uint8_t* bssid)
 {
     WiFiTester& tester = *(WiFiTester*)data;
-    char str_ssid[33];
-    memcpy(str_ssid, ssid, ssid_len);
-    str_ssid[ssid_len] = 0;
-    tester.serialPrint(str_ssid);
-    tester.serialPrint(",");
-    itoa(rssi, str_ssid, 10);
-    tester.serialPrintln(str_ssid);
+    if(ssid != NULL){
```

Comment:
ok got it! Might need to wait for more tests. reading BSSID with no SSID has some hang issue in our testst